### PR TITLE
adds BERT difference plots for colab

### PR DESCRIPTION
**Pull request by 1wheel**

### CODE REVIEW

File: server-side/fill-in-the-blank/scatter-plot-colab/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+### Dev
+
+Hot reload JS
+
+````
+find . | entr rsync -a --omit-dir-times --no-perms --exclude node_modules --exclude .git "$PWD" demo@roadtolarissa.com:../../usr/share/nginx/html/colab/
+````
+
+- https://colab.research.google.com/notebooks/snippets/advanced_outputs.ipynb
```
Comment (jameswex) [inline server-side/fill-in-the-blank/scatter-plot-colab/README.md:11]: will this path be correct when submitted? should it be the github location?

File: server-side/fill-in-the-blank/scatter-plot-colab/spearman-compare/init.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
+console.clear()
+// d3.select('body').selectAppend('div.tooltip.tooltip-hidden')
+
+
+// window.bertProcessedVocab = python_data.vocab
+// var tokenizer = new BertTokenizer()
+// tokenizer.load()
```
Comment (jameswex) [inline server-side/fill-in-the-blank/scatter-plot-colab/spearman-compare/init.js:7]: remove commented-out code.

add license header to all source files

**Review by jameswex — COMMENTED** [review]
some small comments

File: server-side/fill-in-the-blank/scatter-plot-colab/README.md (hunk 1 above)
Comment (1wheel) [reply server-side/fill-in-the-blank/scatter-plot-colab/README.md:17]: Checked in full notebook to LFS and added Dev instructions

File: server-side/fill-in-the-blank/scatter-plot-colab/spearman-compare/init.js (hunk 2 above)
Comment (1wheel) [reply server-side/fill-in-the-blank/scatter-plot-colab/spearman-compare/init.js]: removed my junk headers / added license headers

**Review by jameswex — APPROVED** [review]

**Pull request merged**